plato/draw/pythreejs/Spheres.py

```diff
@@ -15,12 +17,18 @@ def fibonacciPositions(n_b, R=.5):
 class Spheres(draw.Spheres, ThreeJSPrimitive):
     __doc__ = draw.Spheres.__doc__
 
+    _ATTRIBUTES = draw.Spheres._ATTRIBUTES
+
+    _ATTRIBUTES.extend(list(itertools.starmap(ShapeAttribute, [
+        ('vertex_count', np.int32, 64, 0, False,
+         'Number of vertices used to render sphere')
+    ])))
+
     def update_arrays(self):
         if not self._dirty_attributes:
             return
 
-        # TODO make this a configurable attribute
-        vertices = fibonacciPositions(64)
+        vertices = fibonacciPositions(getattr(self, 'vertex_count', 64))
```

Comment:
@bdice Do you need to use `getattr` here instead of just `self.vertex_count`? Off the top of my head, it should be initialized with the value you give in line 23...

Reply:
It wasn't working -- it raised an `AttributeError` and didn't appear to be setting any defaults. I am not sure if I understand why, as I expected `self.vertex_count == 64` would be set by default.

Reply:
Ah, that's because this class (and several others) were missing wrapping themselves in the `ShapeDecorator` that produces the getter and setter methods for those attributes.